integration-testing/src/debugAgentTest/kotlin/PrecompiledDebugProbesTest.kt

```diff
@@ -23,17 +23,16 @@ class PrecompiledDebugProbesTest {
         val binFile = clz.classLoader.getResourceAsStream("DebugProbesKt.bin")!!
         val binContent = binFile.readBytes()
         if (overwrite) {
-            val url = clz.classLoader.getResource("DebugProbesKt.bin")!!
-            val base = url.toExternalForm().toString().removePrefix("jar:file:").substringBefore("/build")
-            val probes = File(base, "jvm/resources/DebugProbesKt.bin")
+            val base = File("kotlin.coroutines.jvm.internal.DebugProbesKt").absolutePath.substringBefore("integration-testing")
+            val probes = File(base, "kotlinx-coroutines-core/jvm/resources/DebugProbesKt.bin")
```

Comment:
I think this is a fine approach, if not the cleanest. It may be surprising that we use the knowledge that this test project is always located in a specific directory of the main project, but it likely won't bite anyone.

A possible alternative would be to replace `overwrite.probes` with another Gradle property that contained the path to `DebugProbesKt.bin`. On the other hand, I think the cost of the hardcode here is fairly low. @qwwdfsad, do we want to preserve the name/function of the Gradle property?